packages/wrangler/src/parse.ts

```diff
@@ -72,6 +72,8 @@ export class ParseError extends UserError implements Message {
 // Therefore, allow particular `ParseError`s to be marked `reportable`.
 export class APIError extends ParseError {
 	#status?: number;
+	code?: number;
```

Comment:
does `code` have to be private as well? And I wonder if the contructor needs to be updated to reflect the new prop
```
constructor({ status, ...rest }: Message & { status?: number }) {
		super(rest);
		this.name = this.constructor.name;
		this.#status = status;
                this.code = code;
	}
```

thinking out loud